src/Configuration/DiactorosConfiguration.php

```diff
@@ -12,28 +18,28 @@ class DiactorosConfiguration implements ConfigurationInterface
     public function apply(Injector $injector)
     {
         $injector->alias(
-            'Psr\Http\Message\RequestInterface',
+            RequestInterface::class,
             // It should not be necessary to force all requests to be server
             // requests, except that Relay uses the wrong type hint:
             // https://github.com/relayphp/Relay.Relay/issues/25
             //
             // 'Zend\Diactoros\Request'
-            'Zend\Diactoros\ServerRequest'
+            ServerRequest::class
```

Comment:
What about this? [RP is still open](https://github.com/relayphp/Relay.Relay/pull/26).